app/code/Magento/CmsGraphQl/Model/Resolver/DataProvider/Page.php

```diff
@@ -44,10 +62,32 @@ public function __construct(
      * @return array
      * @throws NoSuchEntityException
      */
-    public function getData(int $pageId): array
+    public function getDataByPageId(int $pageId): array
```

Comment:
You have added new method `getDataByPageId`, this should be done in a different way - please check this article: https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/
**section** "Introduction of a method to a class or interface"
Check comment below for solution description.

Reply:
**fixed**

Thank you!